### PR TITLE
feat: 🎸 update SubQuery to the latest version

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

update subquery dependency to the latest version. This resolves an issue where bulkUpdate would not work when workers were enabled

### Breaking Changes

BREAKING CHANGE: 🧨 `updated_at` field removed from tables

### JIRA Link

[DA-1236](https://polymesh.atlassian.net/browse/DA-1236)

### Checklist

- [ ] Updated the Readme.md (if required) ?

[DA-1236]: https://polymesh.atlassian.net/browse/DA-1236?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/mappings/entities/mapAsset.ts [hunk 1]
```diff
@@ -164,15 +164,15 @@ export const handleAssetCreated = async (event: SubstrateEvent): Promise<void> =
     rawFundingRoundName ?? api.query.asset.fundingRound(rawTicker),
   ]);
 
-  const name = bytesToString(rawName);
```
Comment (polymath-eric) [inline src/mappings/entities/mapAsset.ts:167]: For some reason `rawName` is now `Codec | Codec`

**Review by prashantasdeveloper — APPROVED** [review]
Approving, but one question -> does this require re-sync as well?

File: tests/unit/serializeLikeHarvester.test.ts [hunk 2]
```diff
@@ -387,32 +387,28 @@ describe('serializeLikeHarvester', () => {
     ).toBe('ticker');
   });
 
-  it('should serialize Calls like the harvester', () => {
+  it.only('should serialize Calls like the harvester', () => {
```
Comment (polymath-eric) [inline tests/unit/serializeLikeHarvester.test.ts:390]:
```suggestion
  it('should serialize Calls like the harvester', () => {
```

**Comment by polymath-eric** [comment]
> Approving, but one question -> does this require re-sync as well?

It needs a resync. The migration code isn't compatible since `updated_at` is required.

**Pull request merged**